tests/python/apps/solana_cmd_builder.py

```diff
@@ -1,11 +1,13 @@
-from typing import List
```

Comment:
can you move all of the stuff in `tests/python` to a separate PR?  it's stuff that LedgerHQ added and i don't think anyone from labs can review it with any confidence.

i would however like to see remote tests added to `tests/src/main.rs`